src/domain/entities/Theme.ts

```diff
@@ -43,6 +44,12 @@ export class Theme {
     public readonly pictures?: {
         [key in ImageSections]?: CellImage;
     };
+    public readonly sharing?: {
```

Comment:
[minor] not sure we need this as optional, as we are setting a default value in the constructor.